Project_FreeRTOS/main/task_rung.h

```diff
@@ -6,8 +6,14 @@
 
 void task_RUNG(void *pvParameters)
 {
+    int rung_noiseThreshold = 50; // cập nhật giá trị
+    int rung_lastValue = 0;
+    int rung_count = 0;
   for(;;)
   {
+    int rung_currentvalue = analogRead(IN_SW_1801P);
+    Serial.print("Rung Value: ");
+    Serial.println(rung_currentvalue); 
    Message sw_send = {0};
       sw_send.id_Tx = OUT_BUZZER;
       sw_send.id_Rx = IN_SW_1801P;
```

Comment:
Please swap values of 2 lines 18 and 19
follow me:
sw_send.id_Tx = IN_SW_1801P;
sw_send.id_Rx = OUT_BUZZER;